src/matter/cluster/GeneralCommissioningCluster.ts

```diff
@@ -55,8 +55,8 @@ export const GeneralCommissioningCluster = Cluster(
     {
         breadcrumb: WritableAttribute(0, UnsignedLongT, BigInt(0)), /* writeAcl: administer */
         commissioningInfo: Attribute(1, BasicCommissioningInfoT),
-        regulatoryConfig: Attribute(2, RegulatoryLocationTypeT),
-        locationCapability: Attribute(3, RegulatoryLocationTypeT),
+        regulatoryConfig: Attribute(2, RegulatoryLocationTypeT), /* default: value of locationCapability */
```

Comment:
How we do stuff like this? Do we need a "constructor method" additionally to the commands for the Cluster Server logics?

Reply:
Hehe ... Also needed to read it multiple times: In fact you can only cover that in code ... it reads enhanced:

The default value of the attribute "regulatoryConfig" is the value of the attribute locationCapability in this cluster :-)

Thats why I meaned that maybe a "constructor" method is helpful that allows a code like

```
if (regulatoryConfig.get() === undefined) {
    regulatoryConfig.set(locationCapability.get()); 
}
```

Reply:
And OnOff cluster would need add Kai kind of a „constructor“
Because it needs to add subscribes to two attribute values because there are timers that need to be adjusted when the value changes.